test/active_record_test.rb

```diff
@@ -7,6 +7,7 @@
 #@formatter:off
 ActiveRecord::Schema.define version: 0 do
   create_table :users, force: true do |t|
+    t.string :bank_account_number
```

Comment:
By adding the `bank_account_number` to the database, won't the decrypted value be saved to the database?